### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by WillsterJohnson** [alexislours/ACNHAPI#34]
I notice that the last commit was [almost a year ago](https://github.com/alexislours/ACNHAPI/commit/6df0d7318a97ab8d0e4a5646bff57180fde1eec5)...

Is this project still being maintained? or is it simply considered complete?

**Comment by alexislours** [comment]
I guess might as well write a public answer since I already got multiple mails regarding this.

Yes, this project is over for me. I have no interest in pursuing it any longer. I'm not sure I want to archive the repository yet but you have better options out there to get data about the game such as [Norviah's npm dependency](https://github.com/Norviah/animal-crossing) and [the spreadsheet from where its data originate](https://docs.google.com/spreadsheets/d/13d_LAJPlxMa_DubPTuirkIV4DERBMXbrWQsmSh8ReK4/edit?usp=sharing).

To go into details as to why: 
- I lost all interest in the game around 5 month after it released but kept updating this project for 6 month past that.
- This project was more intended for personal use for a mobile app I never got around to finish and only kept living this long because it gained traction (the critter part was already up 5 days after release).
- I got spammed with update requests mere minutes each time an update was released by Nintendo by people who didn't even credit me nor ever talked to me. 
- Other projects such as the translation of items I maintained also got used without any credit by some of biggest sites related to the game. I also had some fun releasing some trivia about the game which also got taken out of context by some YouTubers for clicks (without crediting me too for that matter).
- Managing as many different projects on my free time just wasn't doable on the long term for me on top of my job.

tldr: The bad actors in the community got the better of me and simply disgusted me to the point that I don't want to waste anymore of my time on this game or anything that surrounds it.

**Comment by nguyen-anthony** [comment]
@alexislours I feel for you. Thank you for the notice. I'm sorry you had a bad experience with bad eggs. Thank you for the work on this though. I do enjoy using your API for quick data whenever I randomly need it. You're awesome!

**Comment by perryflynn** [comment]
I found this project 10 minutes ago and thought "wtf, awesome!". I can only imagine, how much work it was to collect all these informations. Thank you for that. 🙏
